examples/machinedeployment/machinedeployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: MachineDeployment
+metadata:
+  name: ${CLUSTER_NAME}-md-0
+  labels:
+    cluster.x-k8s.io/cluster-name: ${CLUSTER_NAME}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      cluster.k8s.io/cluster-name: ${CLUSTER_NAME}
+  template:
+    metadata:
+      labels:
+        cluster.k8s.io/cluster-name: ${CLUSTER_NAME}
+    spec:
+      version: v1.14.4
```

Comment:
Should we also template the k8s version?

Reply:
Done!